BUILD

```diff
@@ -37,6 +37,15 @@ cris_cc_library (
     ],
 )
 
+# Note: Only link this to executable directly.
+cris_cc_library (
+    name = "supp",
+    srcs = glob(["src/supp/**/*.cpp"]),
+    hdrs = glob(["src/supp/**/*.h"]),
```

Comment:
Where are these files? I don't think we have them. Have you added them?

Reply:
Oops....